### DISCUSSION TITLE
Android phone scans and gives different Tag ID each time

### DISCUSSION THREAD

**Discussion by abanchieri** [adonno/tagreader#96] [Q&A]
First off,  thanks for the great work. I was able to easily get the tag reader up and running. 

While playing around with it, I found that the reader will also scan my Pixel3A and return a Tag ID to Home Assistant. Each successive scan of the 3A gives a different Tag ID. Any idea on why this happens?

**Comment by zuidwijk** [comment]
> First off,  thanks for the great work. I was able to easily get the tag reader up and running. 
> 
> 
> 
> While playing around with it, I found that the reader will also scan my Pixel3A and return a Tag ID to Home Assistant. Each successive scan of the 3A gives a different Tag ID. Any idea on why this happens? 

Security
